pixiebrix/pixiebrix-extension
# #680: Enable permission grants from editor in Firefox

Blockers:

- [x] https://github.com/pixiebrix/pixiebrix-extension/pull/737#discussion_r666732833
- [x] https://github.com/pixiebrix/pixiebrix-extension/issues/743
- [x] ~~Listen to permissions updated in the extensions page and automatically update the current view~~

How to test it after the blockers:

1. Open website without permission, like https://text.npr.org/
2. Open developer tools
3. Click button:

	<img width="397" alt="Screen Shot" src="https://user-images.githubusercontent.com/1402241/125074968-3586e700-e0e8-11eb-8781-f25b037db44e.png">

4. See options page open
